include/util/typedefs.hpp

```diff
@@ -119,6 +119,14 @@ static const TurnPenalty INVALID_TURN_PENALTY = std::numeric_limits<TurnPenalty>
 static const EdgeDistance INVALID_EDGE_DISTANCE = std::numeric_limits<EdgeDistance>::max();
 static const EdgeDistance INVALID_FALLBACK_SPEED = std::numeric_limits<double>::max();
 
+// Recommended value for passenger vehicles from
+// https://fdotwww.blob.core.windows.net/sitefinity/docs/default-source/content/rail/publications/studies/safety/accelerationresearch.pdf?sfvrsn=716a4bb1_0
+static const double ACCELERATION_ALPHA_CAR = 6.0;
```

Comment:
any reason not to allow passing this value as the parameter in the query rather than matching the string to these constants?

Reply:
is it becasue its hard to describe what the value actually means?

Reply:
These values are just aliases - you can pass `car`, `fast_car`, etc, as strings in the query, or you can pass in an arbitrary number.  I'd like us to be able to adjust these numbers over time if we learn more about how cars behave, without having to have people change their code.